src/dashboard/overview/OverviewItem.tsx

```diff
@@ -1,86 +1,89 @@
-import {
-  createStyles,
-  Divider,
-  Icon,
-  makeStyles,
-  Theme,
-} from "@material-ui/core";
+import { Icon, makeStyles, Theme } from "@material-ui/core";
 import Card from "@material-ui/core/Card";
 import CardContent from "@material-ui/core/CardContent";
 import Grid from "@material-ui/core/Grid";
 import Typography from "@material-ui/core/Typography";
-import GetAppOutlinedIcon from "@material-ui/icons/GetAppOutlined";
 import React, { ReactElement, useState } from "react";
-import { Subject } from "rxjs";
-import api from "../../api";
-import Snackbar from "../../common/components/data-display/feedback/Snackbar";
-import LabeledRow from "../../common/components/data-display/LabeledRow";
-import IconButton from "../../common/components/input/buttons/IconButton";
-import TextButton from "../../common/components/input/buttons/TextButton";
-import { formatDateTimeForFilename } from "../../common/utils/dateUtil";
 import { isServiceReady } from "../../common/utils/serviceUtil";
 import { SERVICES_WITH_ADDITIONAL_INFO } from "../../constants";
 import { Status } from "../../models/Status";
 import ServiceDetails from "./ServiceDetails";
+import {
+  expandIcon,
+  readyIcon,
+  syncingIcon,
+} from "../../common/utils/svgIcons";
 
 export type OverviewItemProps = {
   status: Status;
   opendexdLocked?: boolean;
   opendexdNotReady?: boolean;
 };
 
-const useStyles = makeStyles((theme: Theme) =>
-  createStyles({
-    cardHeader: {
-      padding: theme.spacing(3),
-    },
-    cardContent: {
-      padding: 0,
-      "&:last-child": {
-        paddingBottom: 0,
-      },
-    },
-    statusDot: {
-      height: 10,
-      width: 10,
-      borderRadius: "50%",
-      display: "inline-block",
-      marginRight: 10,
-    },
-    active: {
-      backgroundColor: theme.palette.success.light,
-    },
-    inactive: {
-      backgroundColor: theme.palette.error.light,
-    },
-  })
-);
+const getCardHeaderBackgroundImage = (props: OverviewItemProps) => {
+  if (props.status.status.includes("Ready")) {
+    return "linear-gradient(to right, #1f995a, #00e64d)";
```

Comment:
Maybe we could make it look more like in the design (more blue - it's currently too green). Same applies for the _Ready_ icon.